AISDatabaseParser.cc

```diff
@@ -29,24 +29,14 @@
 #include <stdarg.h>
 
 #include "AISDatabaseParser.h"
-#include "util.h"
 #include "debug.h"
+#include "util.h"
 
 using namespace std;
 
 namespace libdap {
 
-static const not_used char *states[] =
-    {
-        "START",
-        "FINISH",
-        "AIS",
-        "ENTRY",
-        "PRIMARY",
-        "ANCILLARY",
-        "UNKNOWN",
-        "ERROR"
-    };
+static const not_used char *states[] = {"START", "FINISH", "AIS", "ENTRY", "PRIMARY", "ANCILLARY", "UNKNOWN", "ERROR"};
```

Comment:
This bit of auto formatting makes me sad. I find that long lists of values in which each value appears on its own line makes it more readable of me.